index.js

```diff
@@ -42,11 +42,18 @@ function loadRcConfig(callback){
 				var options;
 
 				if(!err) {
+					if (file === "") {
+						file = "{}";
+					}				
 					try {
 						options = JSON.parse(stripJsonComments(file));
 					}
 					catch(e) {
-						err = e;
+						if (options === undefined) {
+							err = new Error("Your .jshintrc file appears to only contain comments. Please add empty curly braces to .jshintrc: {}");
+						} else {
+							err = e;
+						}
```

Comment:
Oh and I think there's no point in printing a stack trace here, it's not gonna be useful in the majority of cases